tests/integration/pipeline.ts

```diff
@@ -478,47 +478,6 @@ describe('Pipeline Editor tests', () => {
     cy.readFile('build/cypress-tests/helloworld.yaml');
   });
 
-  it('should export pipeline as python dsl', () => {
```

Comment:
This is actually testing exporting an airflow pipeline to python dsl, I think this should stay?

Reply:
Ah nice catch! I clearly just searched for `dsl` and went on a removal spree 😄